src/indra_cogex/sources/clinicaltrials/__init__.py

```diff
@@ -95,7 +55,16 @@ def ground_drug(self, drug):
         return None
 
     def get_nodes(self):
-        for index, row in tqdm.tqdm(self.df.iterrows(), total=len(self.df)):
+        nctid_to_data = {}
+        for _, row in tqdm.tqdm(self.df.iterrows(), total=len(self.df)):
+            nctid_to_data[row["NCTId"]] = {
```

Comment:
All the fields, except `NCTId`, need a check for NaN's

Reply:
Resolved in https://github.com/gyorilab/indra_cogex/pull/144/commits/5a9398b1d23e53bf625359933042f6a944b55441